ivy/functional/frontends/scipy/linalg/linalg.py

```diff
@@ -1,6 +1,6 @@
 # global
 import ivy
-from ivy.functional.frontends.scipy.func_wrapper import (
+from ivy.functional.frontends.numpy.func_wrapper import (
     to_ivy_arrays_and_back,
```

Comment:
It would be better practice if you imported the numpy function wrapper into the scipy `func_wrapper` file and used that in the scipy function files.